src/generateMongoidTypes.ts

```diff
@@ -12,12 +14,15 @@ program.requiredOption(
   `./generated/mongoid`
 );
 
-new GeneratorProgram(new MongoidTypeGenerator())
+const preprocessors: Array<Preprocessor> = [new MongoidPreprocessor()];
+
+new GeneratorProgram(MongoidTypeGenerator, preprocessors)
   .generateTypes()
-  .then((result: Array<string>) => {
-    console.log(`Successfully generated ${result.length} types`);
+  .then((result: Array<void>) => {
+    logger.info(`Successfully generated ${result.length} types`);
+    return true;
   })
   .catch((err) => {
-    console.error("ERROR");
-    console.error(err);
+    logger.error("ERROR");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)